src/Nest/Aggregations/Bucket/Terms/TermsAggregation.cs

```diff
@@ -40,7 +40,7 @@ public interface ITermsAggregation : IBucketAggregation
 		TermsAggregationCollectMode? CollectMode { get; set; }
 
 		[JsonProperty("missing")]
-		string Missing { get; set; }
+		object Missing { get; set; }
```

Comment:
Me either, but we'd have to make `AggregationContainer<TFieldType>` as well which opens up a whole can of worms, and I'm not sure we really should.